src/test/java/org/junit/tests/assertion/AssertionTest.java

```diff
@@ -905,6 +906,41 @@ public void expectThrowsUsesCanonicalNameWhenRequiredExceptionNotThrown() {
         fail();
     }
 
+    @Test
+    public void assertContentEqualsPass() throws Exception {
+        CharSequence expected = "StringValue";
+        CharSequence charSequence = new String("StringValue");
+        assertContentEquals(expected,charSequence);
+    }
+
+    @Test
+    public void assertContentEqualsPassBothNull() throws Exception {
+        CharSequence expected = null;
+        CharSequence charSequence = null;
+        assertContentEquals(expected,charSequence);
+    }
+
+    @Test(expected = AssertionError.class)
```

Comment:
Instead of using the "expected" attribute of `@Test` I think the tests should catch the exception and verify that the exception message looks right:

    try {
      assertContentEquals(expected, charSequence);
    } catch (AssertionError e) {
      // make assertions on the message

      return;
    }
    fail("Expected an AssertionError");